lldb/source/Plugins/SymbolFile/DWARF/DWARFASTParserClang.cpp

```diff
@@ -975,6 +975,219 @@ ConvertDWARFCallingConventionToClang(const ParsedDWARFTypeAttributes &attrs) {
   return clang::CC_C;
 }
 
+bool DWARFASTParserClang::ParseObjCMethod(
+    const ObjCLanguage::MethodName &objc_method, const DWARFDIE &die,
+    CompilerType clang_type, const ParsedDWARFTypeAttributes &attrs,
+    bool is_variadic) {
+  SymbolFileDWARF *dwarf = die.GetDWARF();
```

Comment:
I know the previous code was not doing this, but let's take this chance to assert this is not null.

Reply:
Oh I'm surprised to find that `GetDWARF` can in fact return `nullptr`. All of the code in this file assumes that it can't, but not sure what guarantees this...

Reply:
(I think the only way you can realistically can a null pointer here is if you start out with an invalid/empty dwarf die)